### ISSUE TITLE
More informative description of the problem

### ISSUE THREAD

**Issue by volodya-lombrozo** [volodya-lombrozo/jtcop#180]
For now we are printing the next error message:

```
[ERROR] Test FakeMaven.java doesn't have corresponding production class.
[ERROR]         You can either rename or move the test class /Users/lombrozo/Workspace/EOlang/eo/eo-maven-plugin/src/test/java/org/eolang/maven/FakeMaven.java:
[ERROR]         More about that rule you can read here https://github.com/volodya-lombrozo/jtcop/blob/main/docs/rules/all-have-production-class.md
```

It is more or less fine, but it has a lack of suggestions what we can do with the problem. I'm sure, that we have to add information how to suppress a rule or how to solve it, at least in a few words. Moreover, we can create a common "Description" class for all rules.

Also, it would be nice to add codename of the rule to the description

**Comment by 0pdd** [comment]
@volodya-lombrozo 3 puzzles [#182](https://github.com/volodya-lombrozo/jtcop/issues/182), [#183](https://github.com/volodya-lombrozo/jtcop/issues/183), [#184](https://github.com/volodya-lombrozo/jtcop/issues/184) are still not solved.

**Comment by 0pdd** [comment]
@volodya-lombrozo 2 puzzles [#182](https://github.com/volodya-lombrozo/jtcop/issues/182), [#183](https://github.com/volodya-lombrozo/jtcop/issues/183) are still not solved; solved: [#184](https://github.com/volodya-lombrozo/jtcop/issues/184).

**Comment by 0pdd** [comment]
@volodya-lombrozo the puzzle [#183](https://github.com/volodya-lombrozo/jtcop/issues/183) is still not solved; solved: [#182](https://github.com/volodya-lombrozo/jtcop/issues/182), [#184](https://github.com/volodya-lombrozo/jtcop/issues/184).
